### PR TITLE
Update qzoneMoodSpider.py

### PR DESCRIPTION
**Pull request by cats-food**
Hi there! I've quickly looked through your code and found some outdated usages, to make your code more readable, I kindly prompt your to accept my changes.

Kind regards
Yang

### CODE REVIEW

**Comment by Yang-Shiyuan** [comment]
Also, the new-added code will attach extra credits to your work. Since it's the masterpiece created by the Chief Programmer, a.k.a. Dr. Yang. LOL

File: qzoneMoodSpider.py [hunk 1]
```diff
@@ -2,7 +2,9 @@
 """
 Created on Wed Feb 13 13:55:57 2019
 
-@author: xy
+@author: YangShiyuan, all rights reserved.
```
Comment (Yang-Shiyuan) [inline qzoneMoodSpider.py:5]: a

Comment (Yang-Shiyuan) [inline qzoneMoodSpider.py:5]: a

**Review by cats-food — COMMENTED** [review]
a

**Review by tjuxiaoyi — COMMENTED** [review]
nothing useful

**Pull request closed without merging**